src/components/SingleBook.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function SingleBook() {
+  return (
+    <li>
+      Victor is Victory
+      Victor
+      <button type="button"> Remove </button>
+    </li>
+  );
+}
```

Comment:
- [ ] Let's try to make this component takes in a prop and renders the `title` and `author name`. Doing so will make it dynamic somehow.

Reply:
@mmsesay  Thanks for the insight. I believe you are right. I have done it now